src/Orleans.Runtime/ReminderService/InMemoryRemindersTable.cs

```diff
@@ -116,28 +116,24 @@ public bool RemoveRow(GrainReference grainRef, string reminderName, string eTag)
             // assuming the calling grain executes one call at a time, so no need to lock
             if (!reminderTable.TryGetValue(grainRef, out data))
             {
-                logger.Info("1");
```

Comment:
is the reason removing those that logging messaging is garbage?

Reply:
Seems that it is garbage, it just wast left there after initial commit and is not used anywhere, neither informative. Level of these (removed) log messages (Info) is too quite high, everywhere else in this class Trace is used, so I considered it as just debug info which was left by somebody. Actually, seems that all this class InMemoryRemindersTable, and GrainBasedReminderTable ( the only one which instantiates inmemory reminders ) - are not used anywhere in the codebase, so nobody cares about these 1,2,3,4 messages

Reply:
yeah we should remove it .